src/shared/logs-show.c

```diff
@@ -523,6 +523,11 @@ static int output_short(
                 return 0;
         }
 
+        if (identifier && set_contains(j->exclude, identifier)) {
+                log_debug("Skipping manually excluded message.");
```

Comment:
Please drop the logging, otherwise, the functionality of the feature is just replacing matching journal entry.

Reply:
Done